### PR TITLE
New AMIs and update to userdata

### PR DESCRIPTION
**Pull request by paulu-aws**
OpenVPN updated their software which required an additional command in the user data section.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -6,14 +6,10 @@
 	path = submodules/quickstart-compliance-common
 	url = git@github.com:aws-quickstart/quickstart-compliance-common.git
 	branch = master
-[submodule "submodules/quickstart-linux-bastion"]
```
Comment (sshvans) [inline .gitmodules]: Why are we removing the linux bastion?

File: ci/biotech-blueprint-params.t2.json [hunk 2]
```diff
@@ -29,15 +29,7 @@
...
-        "ParameterValue": "$[alfred_getkeypair]"
-    },
-    {   
-        "ParameterKey": "pEC2KeyPairBastion",
-        "ParameterValue": "$[alfred_getkeypair]"
-    },
-    {
-        "ParameterKey": "pBastionAccessCIDR",
-        "ParameterValue": "10.0.0.0/16"
+        "ParameterValue": "paulu-hclscollab-taskcat"
```
Comment (sshvans) [inline ci/biotech-blueprint-params.t2.json]: Seems like you updated the parameter values for your testing, which slipped into the commit. Please fix it.

File: ci/biotech-blueprint-params.t2.json [hunk 3]
```diff
@@ -70,6 +62,14 @@
     {   
         "ParameterKey": "pOpenVPNASAdminPassword",
         "ParameterValue": "Test1234@"
+    },
+    {
+        "ParameterKey": "QSS3BucketName",
+        "ParameterValue": "paulu-hclscollab-taskcat-bucket"
```
Comment (sshvans) [inline ci/biotech-blueprint-params.t2.json]: Seems like you updated the parameter values for your testing, which slipped into the commit. Please fix it.

File: .gitmodules (hunk 1 above)
Comment (paulu-aws) [reply .gitmodules]: We removed the Linux bastion from the stack and replaced it with Systems Management Server Session Manager. I consider it a more secure and audit friendly approach than the Bastion.

File: ci/biotech-blueprint-params.t2.json (hunk 3 above)
Comment (paulu-aws) [reply ci/biotech-blueprint-params.t2.json]: Is there an alfred? reference syntax I should use for QSS3BucketName?

We used ajfriedm-hclscollab-taskcat-bucket before which ive changed it back to.

File: .gitmodules (hunk 1 above)
Comment (sshvans) [reply .gitmodules]: Sounds good!

File: templates/openvpn.template [hunk 4]
```diff
@@ -65,11 +62,11 @@ Mappings:
     AMI:
       OVPN25HVM: OpenVPN-Access-Server-2.5.0-20180501
```
Comment (sshvans) [inline templates/openvpn.template]: Shouldn't this needs to be updated to newer version?

Comment (sshvans) [inline templates/openvpn.template]:
```suggestion
      OVPN25HVM: OpenVPN Access Server 2.6.1-fe8020db-5343-4c43-9e65-5ed4a825c931-ami-0f5d312e085235ed4.4
```

File: ci/biotech-blueprint-params.t2.json (hunk 2 above)
Comment (sshvans) [inline ci/biotech-blueprint-params.t2.json]:
```suggestion
        "ParameterValue": "$[alfred_getkeypair]"
```

File: ci/biotech-blueprint-params.t2.json (hunk 3 above)
Comment (sshvans) [reply ci/biotech-blueprint-params.t2.json]:
```suggestion
        "ParameterValue": "[$taskcat_autobucket]"
```

**Comment by bmoller** [comment]
I've tested this locally after updating input params and it looks good. We should let Jenkins do the promotion to master, so I'm going to close this PR. Thanks @paulu-aws for the updates.

**Pull request closed without merging**